inc/plugin.class.php

```diff
@@ -706,8 +710,17 @@ function clean($ID) {
    **/
    function isActivated($plugin) {
 
-      $activePlugins = $this->getPlugins();
-      return in_array($plugin, $activePlugins);
+      if ($this->isPluginLoaded($plugin)) {
```

Comment:
Only check loaded plugins array will fail if this function is used prior or during initialization prcess (which is done on the core). To fixthis, and kepp the optimization, I put back old check as a fallback check.